test/addOrderProduct.test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+const { addOrder, addOrderProduct, getActiveOrder } = require('../app/models/AddOrderProduct');
+const { assert: { equal, deepEqual, isObject } } = require('chai');
+
+describe('AddOrderProduct model', () => {
+  const newOrder = { orderId: 2, prodId: 2 };
+  const userId = 2;
+  const testOrder = {
+    order_id: 2,
+    customer_id: 1,
+    payment_type: null,
+    order_date: '2017-7-28'
+  }
+
+  describe('addOrder()', () => {
+
+    it('should return an object', () => {
+      return addOrder(userId)
+        .then(data => {
+          isObject(data);
+        })
+    })
+
+    it.skip('should return an id of the last order entered', () => {
+      return addOrder(userId)
+        .then((data) => {
+          deepEqual(data.id, 22);
+        })
+    })
+  })
+
+  describe('addOrderProduct', () => {
+    it('should return an object', () => {
+      return addOrderProduct(userId, newOrder)
+        .then(data => {
+          isObject(data);
+        })
+    })
+
+    it.skip('should return a line_id of the last order_product entered', () => {
```

Comment:
Why are you using skip here?

Reply:
Unless your reset the db, the test will fail after the first run because its looking for a specific 'lastID'